lib/alchemy/controller_actions.rb

```diff
@@ -46,16 +46,7 @@ def alchemy_user_signed_in?
     # Returns the current site.
     #
     def current_alchemy_site
-      @current_alchemy_site ||= begin
-        site_id = params[:site_id] || session[:site_id]
-        if site_id.nil?
-          session.delete :site_id
-          Site.find_for_host(request.host)
-        else
-          session[:site_id] = site_id
-          Site.find(site_id)
-        end
-      end
+      @current_alchemy_site = Site.find(params[:site_id] || session[:site_id] || Site.default.id)
```

Comment:
This changes the complete behavior of site selection in the frontend via the host.
Please don't change this behavior.

I wonder why the tests pass... Maybe because there aren't any. Let me add some.